### PR TITLE
Prefix Sums: Update code to match focus prob

### PR DESCRIPTION
**Pull request by frodakcin**
@HarryWangATX !! Sorry I should have said something earlier!

### CODE REVIEW

**Comment by HarryWangATX** [comment]
LOL no worries

**Review by HarryWangATX — APPROVED** [review]

**Comment by HarryWangATX** [comment]
wait @frodakcin did u add java code?

**Comment by frodakcin** [comment]
> wait @frodakcin did u add java code?

Nope. Do you want to do that?

**Comment by HarryWangATX** [comment]
sure can do that, ill commit to this pr so dont merge yet

**Comment by frodakcin** [comment]
kk thanks!

**Comment by bqi343** [comment]
closes #715

**Comment by HarryWangATX** [comment]
@frodakcin how does this look?

**Comment by HarryWangATX** [comment]
sorry for that, was transcribing code and accidentally made some syntax errrors

**Comment by frodakcin** [comment]
Let's have @andrewwangva review (can't review our own pr)

**Comment by HarryWangATX** [comment]
yea 😭

**Review by HarryWangATX — APPROVED** [review]

File: content/3_Silver/Prefix_Sums.mdx [hunk 1]
```diff
@@ -401,14 +401,60 @@ vl psum(const vl& a) {
...
 
+<JavaSection>
+
+```java
+import java.util.*;
+
+public class Main{
+    static int N, Q;
+    public static void main(String[] args){
+        Scanner in = new Scanner(System.in);
```
Comment (andrewwangva) [inline content/3_Silver/Prefix_Sums.mdx:428]: Why use Scanner (not a big issue but i don't like spreading around the use of Scanner)

**Review by andrewwangva — COMMENTED** [review]
Everything looks p good, I'd say change to BufferedReader but doesn't matter that much.

**Pull request merged**